code/modules/mob/living/simple_animal/slime/subtypes.dm

```diff
@@ -1,79 +1,79 @@
 /mob/living/simple_animal/slime/proc/mutation_table(colour)
-	var/list/slime_mutation[4]
+	var/list/slime_mutation_colors[4]
```

Comment:
I renamed this because there's literally a define on slimes itself called ``var/list/slime_mutation[4]`` and didn't want people to get confused over it.
Slimes really are fucked up creatures.